app/scripts/modules/core/src/application/modal/PermissionsConfigurer.tsx

```diff
@@ -188,8 +192,9 @@ export class PermissionsConfigurer extends React.Component<IPermissionsConfigure
     return (
       <div className="permissions-configurer">
         {this.state.permissionRows.map((row, i) => {
-          const permissionTypeLabel = PermissionsConfigurer.accessTypes.find(type => type.value === row.access).label;
-
+          const permissionTypeLabel = []
+            .concat(PermissionsConfigurer.accessTypes, PermissionsConfigurer.legacyAccessTypes)
+            .find(type => type.value === row.access).label;
```

Comment:
are there cases where we would not expect to find anything and get `undefined`, if so reading `label` could throw an  error

Reply:
No, the label should always be something that is within the `accessTypes` or `legacyAccessTypes`